plugin/adsk/scripts/mayaUsd_USDRootFileRelative.py

```diff
@@ -240,7 +241,12 @@ def updateFilePathPreviewFields(cls, makePathRelative=None):
             # the user is free to enter any extension they want. The layer editor code will then verify
             # (and fix if needed) the file path before saving. We do the same here for preview.
             unresolvedPath = mayaUsdLib.Util.ensureUSDFileExtension(selectedFile) if selectedFile else ''
-            relativePath = mayaUsdLib.Util.getPathRelativeToMayaSceneFile(unresolvedPath) if unresolvedPath else ''
+            """relativePath = mayaUsdLib.Util.getPathRelativeToDirectory(unresolvedPath, cls._parentLayerPath) if unresolvedPath and cls._parentLayerPath != "" else ''"""
```

Comment:
Was this commented code replaced by below? If so it should be removed.

Reply:
removed the comment